packages/central-server/app/tasks/PatientMergeMaintainer.js

```diff
@@ -14,6 +14,7 @@ import {
   mergePatientFieldValues,
   mergePatientProgramRegistrationConditions,
   mergePatientProgramRegistrations,
+  refreshMultiChildRecordsForSync,
```

Comment:
Oh ok now I see why you did it haha...yeah maybe leave it then! Name is slightly confusing but haven't got a better one in mind